pyanaconda/installclasses/installclass_atomic.py

```diff
@@ -0,0 +1,120 @@
+#
+# installclass_atomic.py
+#
+# Atomic-specific partitioning defaults
+#
+# Copyright (C) 2014  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import os
+import shutil
+from pyanaconda.installclasses.fedora import FedoraBaseInstallClass
+from pyanaconda.installclasses.fedora_server import FedoraServerInstallClass
+from pyanaconda.product import productVariant
+from pyanaconda.core import util
+
+import logging
+log = logging.getLogger("anaconda")
+
+__all__ = ['AtomicInstallClass']
+
+
+class AtomicInstallClass(FedoraBaseInstallClass):
+    name = "Atomic Host"
+    stylesheet = "/usr/share/anaconda/pixmaps/atomic/fedora-atomic.css"
+    sortPriority = 11000
+    hidden = False
```

Comment:
Atomic install class can be used in Fedora and RHEL? Otherwise, this line should be removed.

Reply:
Fixed.